sources/packages/web/src/views/PersonalInfoQuestionnaire.vue

```diff
@@ -0,0 +1,167 @@
+        >
+          <RadioButtonList
+            name="residencyQuestion"
+            v-model="personalInfoState.residencySelectedValue"
+            :options="
+              questionnaire.sections.residency.questions
+                .residencyQuestionOptions
+            "
+          >
+          </RadioButtonList>
+        </Question>
+        <Question
+          :text="questionnaire.sections.residency.questions.bcResidencyQuestion"
+        >
+          <RadioButtonList
+            name="bcResidencyQuestion"
+            v-model="personalInfoState.bcResidencySelectedValue"
+            :options="
+              questionnaire.sections.residency.questions
+                .bcResidencyQuestionOptions
+            "
+          >
+          </RadioButtonList>
+        </Question>
+      </Section>
+      <HorizontalSeparator />
+      <Section
+        :title="questionnaire.sections.bankruptcy.title"
+        :sub-title="questionnaire.sections.bankruptcy.subTitle"
+      >
+        <Question
+          :text="questionnaire.sections.bankruptcy.questions.bankruptcyQuestion"
+        >
+          <RadioButtonList
+            name="residencyQuestion"
+            v-model="personalInfoState.bankruptcySelectedValue"
+            :options="
+              questionnaire.sections.bankruptcy.questions
+                .residencyQuestionOptions
+            "
+          >
+          </RadioButtonList>
+        </Question>
+      </Section>
+      <HorizontalSeparator />
+      <Section
+        :title="questionnaire.sections.relationship.title"
+        :sub-title="questionnaire.sections.relationship.subTitle"
+      >
+        <Question
+          :text="
+            questionnaire.sections.relationship.questions.relationshipQuestion
+          "
+        >
+          <RadioButtonList
+            name="relationshipQuestion"
+            v-model="personalInfoState.relationshipSelectedValue"
+            :options="
+              questionnaire.sections.relationship.questions.relationshipOptions
+            "
+          >
+          </RadioButtonList>
+        </Question>
+      </Section>
+    </div>
+    <!-- Only to test the v-model binding, please remove -->
+    <Button @click="printState" class="p-mx-5">Print</Button>
+  </div>
+</template>
+
+<script lang="ts">
+import { reactive } from "vue";
+import Section from "../components/fa-application/Section.vue";
+import Question from "../components/fa-application/Question.vue";
+import RadioButtonList from "../components/fa-application/RadioButtonList.vue";
+import HorizontalSeparator from "../components/fa-application/HorizontalSeparator.vue";
+
+interface PersonalInfoState {
+  residencySelectedValue: string;
+  bcResidencySelectedValue: string;
+  bankruptcySelectedValue: string;
+  relationshipSelectedValue: string;
+}
+
+const yesNoOptions = [
+  { text: "Yes", value: "yes" },
+  { text: "No", value: "no" },
+];
+
+const questionnaire = {
```

Comment:
Seems to me that the questions in this format are becoming harder to read and, since we do not have a requirement to then be dynamic, I am not sure if we should keep them in this format or add the text for "Title", "SubTitle" and "Questions" directly in the components.

Reply:
I always recommend to write all your static text in constants file and bind them with UI but I agree, current structure is complex, I'm adding [technical debt](https://github.com/bcgov/SIMS/issues/188) for this page.